deep_qa/data/tokenizers/word_and_character_tokenizer.py

```diff
@@ -90,13 +91,18 @@ def embed_input(self,
         # done something crazy where you have a word index but your character indices are all zero,
         # you will get a 0 in the mask for that word at the end of this.  But assuming you're using
         # this correctly, you should only get a 0 in the character-level mask in the same places
-        # that you have 0s in the word-level mask, so `Concatenate` will do the right thing.
-        word_encoder = EncoderWrapper(
-                text_trainer._get_encoder(name="word", fallback_behavior="use default params"))
-        # We might need to TimeDistribute this again, if our input has ndim higher than 3.
-        for _ in range(3, K.ndim(characters)):
-            word_encoder = EncoderWrapper(word_encoder, name="timedist_" + word_encoder.name)
-        word_encoding = word_encoder(character_embedding)
+        # that you have 0s in the word-level mask, so `Concatenate` further below will do the right
+        # thing.
+
+        # We want to pass a tensor with shape `(batch_size, word_length, embedding_dim)` to the
```

Comment:
This would be clearer if you start with what the shape actually is and say what we want it to be, rather than this way around.